perfkitbenchmarker/linux_virtual_machine.py

```diff
@@ -314,7 +315,7 @@ def PrepareVMEnvironment(self):
       if self.is_static:
         self.SnapshotPackages()
       self.SetupPackageManager()
-      self.InstallPackages('python')
+      self.Install('python')
```

Comment:
With this change, we may not install the 'python' package through YumInstall, if I've followed the code through. I think this may be okay since we will install a version of Python. To be safe, we may want to add python to the YumInstall alongside the AptInstall.

Reply:
I added "python" to YumInstall alongside what is retrieved by getattr, to be consistent with Apt. I am not adding anything in the situation that getattr returns exactly "python".